start.R

```diff
@@ -191,7 +191,10 @@ if (length(argv) > 0) {
 
 if ("--gamscompile" %in% flags) {
   dir.create(file.path("output", "gamscompile"), recursive = TRUE, showWarnings = FALSE)
-  rungamscompile <- function(tmpModelFile, verbosegamscompile) {
+  rungamscompile <- function(modelFile, cfg, verbosegamscompile) {
```

Comment:
Sorry for not noticing earlier: canonical name would be `runGamsCompile` (or `runGAMSCompile`) and `verboseGamsCompile` (or `verboseGAMSCompile`).